src/ui/hooks/useLocalStorage.ts

```diff
@@ -8,7 +8,11 @@ export function useLocalStorage<T>(key: string, initialValue: T): [T, (_: T) =>
     try {
       const item = window.localStorage.getItem(key);
 
-      return item ? (JSON.parse(item) as T) : initialValue;
+      return item
```

Comment:
Also the things happening here, it is not really clear why these changes are happening and I think they are not part of translating the app?

Reply:
Strings like 'en' were being stored with double quotes because of JSON.stringify. Fixed